integration-mc1-16/src/main/kotlin/org/eln2/utils/OreGen.kt

```diff
@@ -1,44 +1,47 @@
 package org.eln2.utils
 
 import net.minecraft.block.Blocks
+import net.minecraft.util.ResourceLocation
+import net.minecraft.util.registry.Registry
+import net.minecraft.util.registry.WorldGenRegistries
 import net.minecraft.world.gen.GenerationStage
 import net.minecraft.world.gen.feature.ConfiguredFeature
 import net.minecraft.world.gen.feature.Feature
 import net.minecraft.world.gen.feature.OreFeatureConfig
-import net.minecraft.world.gen.feature.template.BlockMatchRuleTest
 import net.minecraftforge.event.world.BiomeLoadingEvent
-import net.minecraftforge.eventbus.api.EventPriority
-import net.minecraftforge.eventbus.api.SubscribeEvent
+import net.minecraftforge.fml.common.Mod
 import org.eln2.ModBlocks
 
 /**
  * Handles ore generation for Eln2
  */
+@Mod.EventBusSubscriber
 object OreGen {
 
     private var ores: List<ConfiguredFeature<*, *>> = mutableListOf()
 
     /**
      * Configures ore generation in the world. Registers against Forge.
      */
+    //var copper_ore = Feature.ORE.withConfiguration(OreFeatureConfig(OreFeatureConfig.FillerBlockType.BASE_STONE_OVERWORLD,ModBlocks.ore_native_copper.block.defaultState,4)).range(64).square().func_242731_b(128)
+
     fun setupOreGeneration() {
          ores = ModBlocks.values().map {
             Feature.ORE.withConfiguration(
-                OreFeatureConfig(BlockMatchRuleTest(Blocks.STONE),
+                OreFeatureConfig(OreFeatureConfig.FillerBlockType.BASE_STONE_OVERWORLD,
                     it.block.defaultState,
                     4
                 )
-            ).range(64).square().func_242731_b(60000)
+            ).range(64).square().func_242731_b(120)
         }
+            //Registry.register(WorldGenRegistries.CONFIGURED_FEATURE, ResourceLocation(org.eln2.MODID,"ore_native_copper"), copper_ore)
     }
 
-    @SubscribeEvent(priority = EventPriority.HIGHEST)
-    fun gen(event: BiomeLoadingEvent) {
```

Comment:
Welp, that tutorial lied to me...

Reply:
I tried several configurations until it worked... The most notorious error was the 60000, almost every stone block was replaced with copper ore, then, it was replaced with flubber block... Lower it to 120 seems to do the job in decrease the amount and the placement

Reply:
Yeah, I set it to a ridiculous number so that it would replace all of the stone with copper so that I would know if it worked :D